solr/src/main/conf/schema.xml

```diff
@@ -0,0 +1,613 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
is this copied verbatim from somewhere? if not, it should follow the header format we use rather than that for contributing to an ASF project.

Also, should this be in src/main/resources? Where do we expect it to end up in the solr-specific convenience binary tarball?